FerramAerospaceResearch/FARAeroComponents/FARVesselAero.cs

```diff
@@ -274,14 +274,14 @@ private void CalculateAndApplyVesselAeroProperties()
 
             float pseudoKnudsenNumber = (float)(machNumber / (reynoldsNumber + machNumber));
 
-            Vector3 frameVel = Krakensbane.GetFrameVelocityV3f();
 
             //if(_updateQueued)       //only happens if we have an voxelization scheduled, then we need to check for null
             for (int i = _currentAeroModules.Count - 1; i >= 0; i--)        //start from the top and come down to improve performance if it needs to remove anything
             {
-                FARAeroPartModule m = _currentAeroModules[i];
-                if (m != null && m.part != null && m.part.partTransform != null)
-                    m.UpdateVelocityAndAngVelocity(frameVel);
+                if (_currentAeroModules[i] != null && _currentAeroModules[i].part != null && _currentAeroModules[i].part.partTransform != null)
+                {
+                    _currentAeroModules[i].UpdateVelocityAndAngVelocity(Krakensbane.GetFrameVelocityV3f());
```

Comment:
https://github.com/dkavolis/Ferram-Aerospace-Research/blob/a48e71656689b08f999c52de82ef48120791508f/FerramAerospaceResearch/FARAeroComponents/FARVesselAero.cs#L281-L284

`frameVel` should probably stay so that `Krakensbane.GetFrameVelocityV3f()` is not called multiple times in the loop